conda-store/environment.yaml

```diff
@@ -2,19 +2,14 @@ name: conda-store
 channels:
   - conda-forge
 dependencies:
-  - pip
   - jupyterhub<2.0.0
-  - jupyterlab>=4.0.0
+  - jupyterlab>4.0
   - nb_conda_kernels
-  # - nb_conda_store_kernels >=0.1.4
-  - nodejs=16
+  - nodejs=18
   - yarn
-  # conda-store client dependencies
-  - yarl
-  - aiohttp>=3.8.1
-  - rich
-  - click
-  - ruamel.yaml
+  # installer
+  - constructor # runtime dep but must be conda installed
+  - pip
   - pip:
       - https://github.com/yuvipanda/jupyter-launcher-shortcuts/archive/refs/heads/master.zip
```

Comment:
This should be _at least_ a commit hash. But ideally we switch to a released (conda) package.

Reply:
I do not know why this is even here 🤷🏽 will replace with the released package and see how it goes